appshell/appshell_extensions.cpp

```diff
@@ -842,6 +842,8 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
             uberDict->SetList(0, dirContents);
             uberDict->SetList(1, allStats);
             responseArgs->SetList(2, uberDict);
+        } else if (message_name == "GetRemoteDebuggingPort") {
```

Comment:
We're changing the normal function to one that takes callback? Does this have any impact on existing usages? This is a breaking change, though I guess no extensions will ideally use this.

Reply:
Yes, it does impact one use case, and fixed that with commit: https://github.com/adobe/brackets/pull/14956/commits/8677756aa443b98523852f86bfcbfc1a77d93478